src/styles/theme/default.theme.ts

```diff
@@ -137,6 +138,7 @@ const theme: OakTheme = {
     pink: "black",
     pink50: "black",
     pink30: "black",
+    pink60: "black",
```

Comment:
isn't this rather strange ?

Reply:
from what I gather this is used to decide whether an icon will have a white or black fill against whatever colour background, but could def use some clarity!